tests/unit/CoreQuoteTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace WPGraphQL\ContentBlocks\Unit;
+
+final class CoreQuoteTest extends PluginTestCase {
+	/**
+	 * The ID of the post created for the test.
+	 *
+	 * @var int
+	 */
+	public $post_id;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->post_id = wp_insert_post(
+			[
+				'post_title'   => 'Post with Quote',
+				'post_content' => '',
+				'post_status'  => 'publish',
+			]
+		);
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function tearDown(): void {
+		parent::tearDown();
+
+		wp_delete_post( $this->post_id, true );
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function query(): string {
+		return '
+			fragment CoreQuoteBlockFragment on CoreQuote {
+				attributes {
+					anchor
+					backgroundColor
+					citation
+					className
+					cssClassName
+					fontFamily
+					fontSize
+					gradient
+					# layout
```

Comment:
Why are this and `textAlign` commented out?

Reply:
<img width="671" alt="Screenshot 2024-09-23 at 9 15 35 PM" src="https://github.com/user-attachments/assets/c10c759a-7891-4a9d-a2b3-9f504474bc8d">
<img width="953" alt="Screenshot 2024-09-23 at 9 16 02 PM" src="https://github.com/user-attachments/assets/e73522c0-aaa6-4a97-9b90-c328d51aee7f">

Reply:
What version of WP?
![alt text](image.png)

Reply:
My local is WP 6.6.2

Reply:
I'm assuming you're running PHPUnit in the docker container.

Either way, looks like the issue is that in 6.6 < the fields don't exist. I'll take a quick stab and if not leave it for you in the morning.

Reply:
I ran out of time. To handle these, check the [slack instructions](https://rtcamp.slack.com/archives/CK2CEH4F6/p1727126999500269?thread_ts=1726011552.525109&cid=CK2CEH4F6)